src/services/balancer/gauges/gauges-subgraph.client.ts

```diff
@@ -21,7 +21,7 @@ export class GaugesSubgraphClient {
   }
 
   public payloadFor(query) {
-    return JSON.stringify({ query: jsonToGraphQLQuery({ query }) });
```

Comment:
I assume this is just redundant? If not will it not break the request?

Reply:
I think so, also it was preventing MSW from detecting this requests as Graphql ones.